src/electeeManagement/views.py

```diff
@@ -28,9 +28,42 @@ def all_electees(request):
     requirements = dict(
         (requirements.requirement, requirements) for requirements in Requirements.objects.all())
 
+    # process the electee progress, so that wo don't have to run those if-else in template html
+    print type(electee_list)
+    electee_list_plain = []
+    for electee in electee_list:
+        progress = {}
+        progress['uniqname'] = electee.member.uniqname
+        progress['first_name'] = electee.member.first_name
+        progress['last_name'] = electee.member.last_name
+        is_undergraduate = electee.member.is_undergraduate()
+
+        progress['num_socials_approved'] = electee.num_socials_approved
+        progress['num_service_hours_approved'] = electee.num_service_hours_approved
+        progress['electee_interview'] = electee.electee_interview
+        progress['electee_exam'] = electee.electee_exam
+        progress['dues'] = electee.dues
+
+        # check for if requirement meets
+        req_social = 'A_UG_SOCIAL' if is_undergraduate else 'B_G_SOCIAL'
+        req_service = 'C_UG_TOTAL_HOURS' if is_undergraduate else 'D_G_TOTAL_HOURS'
```

Comment:
You could remove the need for the is_undergraduate variable entirely if you just did 

`req_social, req_service = ('A_UG_SOCIAL', 'C_UG_TOTAL_HOURS') if electee.member.is_undergraduate() else ('B_G_SOCIAL', 'D_G_TOTAL_HOURS')`

Again, I'm just feeling that this code is unnecessarily verbose.